zerver/data_import/import_util.py

```diff
@@ -572,19 +572,19 @@ def write_avatar_png(avatar_folder: str,
 ListJobData = TypeVar('ListJobData')
 def run_parallel_wrapper(f: Callable[[ListJobData], None], full_items: List[ListJobData],
                          threads: int=6) -> Iterable[Tuple[int, List[ListJobData]]]:
-    logging.info("Distributing %s items across %s threads" % (len(full_items), threads))
+    logging.info("Distributing %s items across %s threads", len(full_items), threads)
 
     def wrapping_function(items: List[ListJobData]) -> int:
         count = 0
         for item in items:
             try:
                 f(item)
             except Exception:
-                logging.info("Error processing item: %s" % (item,))
+                logging.info("Error processing item: %s", item)
                 traceback.print_exc()
             count += 1
             if count % 1000 == 0:
-                logging.info("A download thread finished %s items" % (count,))
```

Comment:
Are we sure we don't want to use f-strings for messages like this?  More discussion here: https://chat.zulip.org/#narrow/stream/3-backend/topic/f-strings/near/866745

Reply:
`logging.info("A download thread finished %s items", count)` is a more idiomatic use of the `logging` API, and it knows to skip the formatting operation if the logging level is greater than `INFO`.

Reply:
Another subtle difference is that if formatting fails, `logging` logs the exception but continues running:

```python
logging.warn("%(blue)s", {"red": 1, "green": 2})
# logs KeyError and continues execution here

logging.warn("%(blue)s" % {"red": 1, "green": 2})
# raises KeyError and does not continue execution here
```